sparkles/tests/test_checks.py

```diff
@@ -148,6 +149,28 @@ def test_guide_count_er4():
         {'text': 'ER with 6 guides but 8 were requested', 'category': 'caution'}]
 
 
+def test_include_exclude():
+    """Test INFO statement for explicitly included/excluded entries"""
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=8, mag=np.linspace(7.0, 8.75, 8))
+    aca = get_aca_catalog(**mod_std_info(obsid=50000, n_fid=0, n_acq=5, n_guide=8),
+                          stars=stars, dark=DARK40,
+                          exclude_ids_acq=100,
+                          include_ids_guide=107,
+                          exclude_ids_guide=[100, 101],
+                          include_ids_acq=[106, 107],
+                          include_halfws_acq=[140, 120],
```

Comment:
I've forgotten, for the implicit case where one wants to include an acq but let proseco optimize halfw, does that end up passed from ORViewer as 0 or None?  And what do we want to see for INFO for those case?  Or doesn't matter/don't care?

Reply:
I think we will get a single 0, but I'm not certain (though I'm pretty confident it won't be `None`).  

The upshot is that whatever gets passed (including the argument not being supplied at all), we will be able to unambiguously interpret the output without trouble.  In other words, these new info statements provide a complete record of all include/exclude arguments via `call_args`, and we the reviewers understand the implications.